src/progress-indicator/progress-indicator.style.tsx

```diff
@@ -109,22 +52,24 @@ export const IndicatorBar = styled.div<IndicatorProps>`
             background-color: ${color(props)};
         `;
     }};
-
-    ${MediaQuery.MaxWidth.mobileL} {
-        width: ${INDICATOR_BAR_WIDTH_MOBILE + "rem"};
-    }
 `;
 
-export const IndicatorTitle = styled(Text.BodySmall)<IndicatorProps>`
-    margin-right: 0.5rem;
-
+export const IndicatorTitleDesktop = styled(Text.BodySmall)<IndicatorProps>`
+    overflow-wrap: anywhere;
     ${(props) => {
         const { highlighted } = props;
         const color = highlighted ? Color.Primary : Color.Neutral[4];
```

Comment:
Figma shows N3 instead of N4, can confirm with UX again?

Reply:
> The colour should be N3 as per figma. N4 on our white background fails colour contrast/accessibility standards.

Verified with UX, will update the change 👍